### DISCUSSION TITLE
Progetto Smart home

### DISCUSSION THREAD

**Discussion by Andrececca** [indomus/forum#3189] [Nuove idee, proposte, progetti e guide]
Ciao a tutti, sono qui per chiedere pareri e idee per il progetto di realizzare una Smart home. Premetto che ho acquistato una casa con impianto elettrico tradizionale che non subirà interventi a parte giusto qualche interruttore in più o comuqnue poca roba. La mia  idea è quella di rendere Smart questa abitazione, un po' ne capisco ma è un mondo nuovo per me, per questo chiedo aiuto per quanto riguarda la scelta dei componenti. Mi sono un po' documentato e pensavo di fare così, il tutto comandato da un tablet a parete con home assistant.

- tablet (probabilmente Android per via del prezzo)
- Raspberry Pi 4 Model B
- SONOFF Zigbee Bridge Pro (per comandare i vari switch zigbee)
- sonoff mini zigbee (per comandare i punti luci della casa)
- sonoff dual r3 (per tapparelle elettriche)
- Termostato (ancora da definire)
- Vari elettrodomestici

Ora, al mia scelta cade su zignerò poiché io la connessione Internet vorrei sfruttarla al massimo per giocare online e ho paura che mettendo interruttori con protocollo Wi-Fi mi vadano a rallentare la connessione utilizzata per giocare.

cosa ne pensate delle mie scelte?

**Comment by indomus** [comment]
> cosa ne pensate delle mie scelte?

che forse sono un po' rivedibili:

- raspberry? scegli Mini PC. ora come ora non ha più senso, raspberry, in questo contesto;
- SONOFF Zigbee Bridge Pro? preferisci sonoff zbdongle-e collegato all'host (salvo tu non abbia specifiche esigenze);
- sonoff dual r3? valuta gli shelly 2.5
